src/components/heading/heading.pw.tsx

```diff
@@ -121,6 +121,28 @@ test.describe("Heading component", () => {
     });
   });
 
+  testData.forEach((backLink) => {
```

Comment:
comment: I'm not sure what the value of this .forEach() is

I know you're just basing it off whatever was already there, but it doesn't actually seem to loop through anything so I think it would be better not to be a .forEach()